src/main/resources/blockchain/default-genesis.json

```diff
@@ -0,0 +1,8905 @@
+{
+  "difficulty": "0x0400000000",
+  "extraData": "0x11bbe8db4e347b4e8c937c1c8370e4b5ed33adb3db69cbdb7a38e1e50b1b82fa",
+  "gasLimit": "0x1388",
+  "nonce": "0x0000000000000042",
+  "ommersHash": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+  "timestamp": "0x0",
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
> are you sure xor, and, not are arithmetic operations? ;)

I should have said _algebraic_, but yes. You apply this algebra to numbers and you get a number.

> associated information to treat them as characters

And in our code we have information what to treat the mixHash as. And it's textual representation doesn't need to depend on that.